SeaPublicWebsite.BusinessLogic/Models/Enums/CavityWallsInsulated.cs

```diff
@@ -4,13 +4,13 @@ namespace SeaPublicWebsite.BusinessLogic.Models.Enums
 {
     public enum CavityWallsInsulated
     {
-        [GovUkRadioCheckboxLabelText(Text = "I'm not sure")]
-        DoNotKnow, 
         [GovUkRadioCheckboxLabelText(Text = "Yes, they are all insulated")]
         All,
         [GovUkRadioCheckboxLabelText(Text = "Some are insulated and some not")]
         Some,
         [GovUkRadioCheckboxLabelText(Text = "No, they are not insulated")]
         No,
+        [GovUkRadioCheckboxLabelText(Text = "I don't know")]
+        DoNotKnow, 
```

Comment:
Reordered all questions to have DoNotKnow at the end. Will let the team know to not use previous reference nubmers.